WEB/CSS/demo_style.css

```diff
@@ -1,2 +1,61 @@
 /*THIS FILE CONTAINS THE STYLES FOR THE PAGE "DEMO". 
-LOOK AT STYLE.CSS TO SEE THE SETTINGS COMMON TO ALL PAGES*/
\ No newline at end of file
+LOOK AT STYLE.CSS TO SEE THE SETTINGS COMMON TO ALL PAGES*/
+
+/* drag and drop styles */
+.dragDrop {
```

Comment:
Fiddling with the demo page on different page sizes, I found out that the `dragDrop` is in front of the fixed menu bar from up top, as illustrated in the pictures attached. I have done some searching and using `z-index` might help.

![img1](https://user-images.githubusercontent.com/13762182/38462989-1c3c2362-3af8-11e8-9026-d3f2fabf83e6.png)
![img2](https://user-images.githubusercontent.com/13762182/38462991-1c5adb36-3af8-11e8-8b7f-fb792fa8e3a9.png)